bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/config/ShellyThingConfiguration.java

```diff
@@ -45,4 +45,5 @@ public class ShellyThingConfiguration {
     public String serviceName = "";
 
     public Boolean enableBluGateway = false;
+    public Boolean enableRangeExtender = true;
```

Comment:
prefer `boolean`

Reply:
same, maybe null

Reply:
Where is set to null, i couldn't find it? If it can be null, i would prefer to make it a @Nullable boolean 
(same with enableBluGateway, but i can understand if you want that out of this PR scope)